tests/test_zcl_clusters.py

```diff
@@ -1,11 +1,21 @@
+from __future__ import annotations
+
 import asyncio
+from datetime import datetime, timezone
 import re
+from typing import Any
 
 import pytest
 
+try:
+    from zoneinfo import ZoneInfo
+except ImportError:
+    from backports.zoneinfo import ZoneInfo  # type: ignore[no-redef]
```

Comment:
I guess we can also remove this then?

Reply:
Ah, more from a bad merge. Good point. We can drop 3.8 from the main CI.